psydac/api/postprocessing.py

```diff
@@ -1311,10 +1311,12 @@ def _load_fields_helper(self, patch_name, space_name, field_name, coeff, contain
                 V = space.spaces[i].vector_space
                 index_coeff = tuple(slice(s, e + 1) for s, e in zip(V.starts, V.ends))
                 field.coeffs[i][index_coeff] = coeff[i][index_coeff]
+                field.coeffs[i].update_ghost_regions()
         else:
             V = space.vector_space
             index_coeff = tuple(slice(s, e + 1) for s, e in zip(V.starts, V.ends))
             field.coeffs[index_coeff] = coeff[index_coeff]
+            field.coeffs.update_ghotst_regions()
```

Comment:
Typo!

Reply:
Fixed in [18a090c](https://github.com/pyccel/psydac/pull/234/commits/18a090c4864587c5a3f951cfdec613fedbc124d4)